src/VecSim/vec_sim.h

```diff
@@ -55,6 +55,30 @@ int VecSimIndex_DeleteVector(VecSimIndex *index, size_t id);
  */
 double VecSimIndex_GetDistanceFrom(VecSimIndex *index, size_t id, const void *blob);
 
+/**
+ * @brief Calculate the distance of a vector from an index to a vector.
+ * @param index the index from which the first vector is located, and that defines the distance
+ * metric.
+ * @param id the id of the vector in the index.
+ * @param blob binary representation of the second vector. Blob size should match the index data
+ * type and dimension.
+ * @param normalize flag for skipping normalization. useful when you intend to use this function
+ * many times, and your blob is already normalized.
+ * @return The distance (according to the index's distance metric) between `blob` and the vector
+ * with id `id`.
+ */
+double VecSimIndex_GetDistanceFrom_withFlag(VecSimIndex *index, size_t id, const void *blob,
+                                            bool normalize);
```

Comment:
When `normalize=True`, this API is equivalent to `VecSimIndex_GetDistanceFrom`, right? 
So why not calling it `VecSimIndex_GetDistanceFromNormalizedVector` (which means that we assume that `blob` is normalised) and avoid the `normalize` flag?

Reply:
this way Ill need to write the function twice or use helper func. not a big deal just another approach :). 
@DvirDukhan ?